examples/sim_params.json

```diff
@@ -0,0 +1,16 @@
+{
+  "list_of_channels_under_test": [1, 18, 37, 56, 75],
+  "raman_parameters": {
+    "flag_raman": true,
+    "space_resolution": 10e3,
+    "tolerance": 1e-8,
+    "verbose": 2
```

Comment:
If this is about a debugging verbosity, let's control this via an option to the program (and an appropriate log level).